legend-depot-artifacts-repository-api/src/main/java/org/finos/legend/depot/artifacts/repository/services/RepositoryServices.java

```diff
@@ -73,7 +73,8 @@ public List<VersionMismatch> findVersionsMismatches()
         {
             try
             {
-                final List<String> storeVersions = projects.getVersions(p.getGroupId(), p.getArtifactId());
+                final ArrayList<String> storeVersions = new ArrayList<>(projects.getVersions(p.getGroupId(), p.getArtifactId()));
```

Comment:
why do you just get all project versions not that the strings.... so you can figure whats ecluded or not